src/descriptors/power_system_structs.json

```diff
@@ -615,14 +636,15 @@
       },
       {
         "name": "internal",
+        "comment": "power system internal reference, do not modify",
         "data_type": "InfrastructureSystemsInternal"
       }
     ],
     "supertype": "ACBranch"
   },
   {
     "struct_name": "Transformer2W",
-    "docstring": "\n    mutable struct Transformer2W <: ACBranch\n        name::String\n        available::Bool\n        activepower_flow::Float64\n        reactivepower_flow::Float64\n        arc::Arc\n        r::Float64  # System per-unit value\n        x::Float64\n        primaryshunt::Float64\n        rate::Union{Nothing, Float64}\n        _forecasts::InfrastructureSystems.Forecasts\n        internal::InfrastructureSystemsInternal\n    end\n\nThe 2-W transformer model uses an equivalent circuit assuming the\nimpedance is on the High Voltage Side of the transformer. The model\nallocates the iron losses and magnetizing susceptance to the primary side.\n\n# Arguments\n\n-`r::Float64`: System per-unit\n-`valuex::Float64`: System per-unit value\n-`primaryshunt::Float64`: System per-unit value\n-`internal::InfrastructureSystemsInternal`: Power System internal reference, do not modify\n",
+    "docstring": "The 2-W transformer model uses an equivalent circuit assuming the\nimpedance is on the High Voltage Side of the transformer. The model\nallocates the iron losses and magnetizing susceptance to the primary side.",
```

Comment:
I think it would be better to leave line breaks out. If they are needed then the auto-generation code should handle it.